### ISSUE TITLE
Standalone Battery Check

### ISSUE THREAD

**Issue by albertofralbe** [dalathegreat/Battery-Emulator#233]
Hi, I was wondering if it's possible to diagnose the battery BMS, check the SOH and the cells voltage without an inverter attached as I'm about to acquire a battery to go through with this amazing project. Thank you!

**Comment by ffundoxx** [comment]
Yes, it is - and this is definitely something that you should ask the battery seller to provide to you, a full battery health check. If you're buying from e.g. a salvage station or car service facility, they usually provide this as a proof that the battery works. There's little benefit in buying a non-working battery, unless you want to service it yourself.
You can check the battery status with e.g. a OBD reader, as long as you can power up the battery for its BMS to run (usually a 12V system), and have a port to read the data from. It doesn't need to be connected to an inverter at that point.
